app/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ if(WIN32)
 else()
   add_executable(
     app
+    ""
```

Comment:
Is this empty string supposed to be here?

Reply:
I will fix it. I add it coz old cmake versions don't support `add_executable` with the target only.